xml/ubl21/example/ubl21_invoice2.xml

```diff
@@ -0,0 +1,58 @@
+<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
```

Comment:
same here let's just name it `invoice2.xml`
or we can have examples `short-invoice.xml` and `long-invoice.xml`